sdcflows/workflows/apply/registration.py

```diff
@@ -133,14 +138,17 @@ def init_coeff2epi_wf(
             ("target_mask", f"moving_image_mask{mask_trait_s}"),
             ("fmap_mask", f"fixed_image_mask{mask_trait_s}"),
         ]),
-        (coregister, outputnode, [("warped_image", "target_ref")]),
+        (coregister, outputnode, [
+            ("warped_image", "target_ref"),
```

Comment:
This would require changing line 123 to `True`, unless you only want the reference when debugging.

Reply:
That's fine. That's not new, all I want for now is the transform.